### ISSUE TITLE
TA Feedback (milestone 2): Plots & tables

### ISSUE THREAD

**Issue by jbourak** [UBC-MDS/DSCI_522_group27#78]
- The correlation matrix should be generated using only the training data, not the entire data set (this is essentially EDA). 
  - In fact, I'm not sure whether this figure is useful to the reader at all. If you did not use any information that you gathered from it in your analysis, there is no need to include it.
- It is quite hard to digest all the numbers in table 1: does the reader need to see all of these? A plot showing the effect of hyper-parameters on a metric of your choice might be more effective--were there other hyper-parameters that gave comparable scores?
- What is the importance of figure 3? Did you use it to make any decisions or adjustments to your model? If not, you may consider removing it. Or, draw some conclusions from it, explain them, and include these in your analysis!
